frontend/src/components/story/StoryCard.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import Tag from "./Tag";
+import "./StoryCard.css";
+
+type StoryCardProps = {
+  id: number;
+  title: string;
+  description: string;
+  youtubeLink: string;
+  level: number;
+};
+
+const StoryCard = ({
+  id,
+  title,
+  description,
+  youtubeLink,
+  level,
+}: StoryCardProps) => {
+  const embedLink = (originalYoutubeLink: string): string => {
+    /*
+    Should take the youtube link and transform it into the form
```

Comment:
wording nit: "Transforms originalYoutubeLink into embed link appropriate for iframe" or something along these lines. It's more imperative and refers to the parameters by name